packages/zod-nestjs/src/lib/create-zod-dto.ts

```diff
@@ -85,5 +94,5 @@ export const createZodDto = <T extends OpenApiZodAny>(
     }
   }
 
-  return SchemaHolderClass;
+  return SchemaHolderClass as unknown as ZodDtoStatic<T>;
```

Comment:
Need to coerce this now with the changes to `CompatibleZodInfer`.

Reply:
Is there way not to coerce? 
I realize that TS can need this. Every time we do it, we are breaking type safety.